tree/ntuple/v7/src/RColumn.cxx

```diff
@@ -27,10 +27,9 @@ ROOT::Experimental::Internal::RColumn::RColumn(EColumnType type, std::uint32_t c
                                                std::uint16_t representationIndex)
    : fType(type), fIndex(columnIndex), fRepresentationIndex(representationIndex), fTeam({this})
 {
-   // TODO(jblomer): fix for column types with configurable bit length once available
    const auto [minBits, maxBits] = RColumnElementBase::GetValidBitRange(type);
-   assert(minBits == maxBits);
-   fBitsOnStorage = minBits;
+   (void)minBits;
```

Comment:
@jblomer the way I implemented the variable bit length seems to not need this information here, but maybe I'm missing something and/or there is a better way to implement it that you were foreseeing when writing this